### ISSUE TITLE
Path not find in Managed (.NET/C#) mode

### ISSUE THREAD

**Issue by yegord69** [UnamSanctam/UnamBinder#29]
Hi @UnamSanctam !

When I try to build two files in the same directory as the binder I get an error that the file on the given path is missing. Same issue happens if I entry the absolute path of the file. The issue only happens in Managed (.NET/C# mode). It works with the Native one. 

Can you please help me in resolving this issue?

**Comment by UnamSanctam** [comment]
Does it work if you move them to another location?

**Comment by yegord69** [comment]
Nope

**Comment by UnamSanctam** [comment]
Try running the binder as administrator.

**Comment by yegord69** [comment]
Nope it doesn't work, and the error always occurs for the 2nd file. (2 files added)

**Comment by UnamSanctam** [comment]
Try redownloading the binder here https://github.com/UnamSanctam/UnamBinder/releases and see if that works.

**Comment by yegord69** [comment]
No it doesn't, I am using the latest version.

**Comment by UnamSanctam** [comment]
Hmm I uploaded a hotfix for some bug like that I think 17 days ago, I just bound 5 files together with the latest one.

**Comment by tarekxxx** [comment]
have same problem in vm

**Comment by UnamSanctam** [comment]
> have same problem in vm

Sorry for the late answer, I was away from home for a bit. Try redownloading the latest build from the "releases" page.

**Comment by Riddle1337** [comment]
> > have same problem in vm
> 
> Sorry for the late answer, I was away from home for a bit. Try redownloading the latest build from the "releases" page.

The compiled build works. Are you planning on uploading the source code for the newest build? The source code that is attached to that release is code with the bug still in it.

**Comment by UnamSanctam** [comment]
> The compiled build works. Are you planning on uploading the source code for the newest build? The source code that is attached to that release is code with the bug still in it.

Yes I just don't want to release a new version with only one line of code changed so I was trying to think of other things to add or change for it.
The change is moving this line:
https://github.com/UnamSanctam/UnamBinder/blob/b79b6f3eb56bd417e089da35393efa4396f31163/UnamBinder/Forms/Builder.cs#L75
into the for loop right beneath it on line 80.

**Comment by Riddle1337** [comment]
> > The compiled build works. Are you planning on uploading the source code for the newest build? The source code that is attached to that release is code with the bug still in it.
> 
> Yes I just don't want to release a new version with only one line of code changed so I was trying to think of other things to add or change for it. The change is moving this line:
> 
> https://github.com/UnamSanctam/UnamBinder/blob/b79b6f3eb56bd417e089da35393efa4396f31163/UnamBinder/Forms/Builder.cs#L75
> 
> 
> into the for loop right beneath it on line 80.

Completely makes sense not wanting to release a whole new version for a 1 line change. Thanks very much for the fix!
